nixos/modules/services/web-servers/bluemap.nix

```
@@ -70,9 +70,8 @@ in {
     };
 
     host = mkOption {
-      type = lib.types.str;
-      default = "bluemap.${config.networking.domain}";
-      defaultText = lib.literalExpression "bluemap.\${config.networking.domain}";
+      type = lib.types.nullOr lib.types.str;
+      default = null;
```

```suggestion
      type = lib.types.str;
```

I'd be fine with just keeping it undefined and letting the module system complain if it's needed

---

Yeah, good point. If null is not valid anyway, we should not allow in the type.

---

I dropped the default and kept it typed as str.